profiles/base.yaml

```diff
@@ -1452,9 +1452,13 @@ tarantula_debug: false
 # Number of minutes to wait between performing a verb on an item.
 faux_atmo_interval: 15
 
+# Set to true to randomly select an item and verb to perform.
```

Comment:
To clarify what I was suggesting: I meant to randomize the starting item, then round robin.

The current implementation always picks randomly. Mine is a weak preference, so let me know if you want to keep this (and I'll merge the PR) or change it.

Reply:
What do you think of if `@random = true` then during init the items and their verbs are shuffled? From then on it's round-robin to avoid the nature of random-every-time having chance of you performing the same verb/item too soon or ignoring items.